modules/osd-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/proxies/configuring-cluster-wide-proxy.adoc
+
+[id="osd-cluster-wide-proxy-config_{context}"]
+= Configuring a cluster-wide proxy using {product-title}
+
+You can add a proxy to your cluster when you are creating a new one. Prior to installation, however, you should verify that the proxy is accessible from their intended cluster virtual private cloud (VPC) and its private subnets.
+
+.Procedure
+To create a cluster with a proxy, run the following command:
+[source, terminal]
+----
+ocm create cluster \
+ <other arguments here> \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
+
+[id="osd-proxy-considerations"]
+== Considerations
+
+1. The `http-proxy`, `https-proxy`, and `additional-trust-bundle-file` arguments are all optional.
+1. If you use the `additional-trust-bundle-file` option without an `http(s)-proxy` argument, the passed additional trust bundle will be set on the cluster, but it will not be configured to be used with the proxy.
+1. The `additional-trust-bundle-file` argument is a file path pointing to a bundle of Privacy Enhanced Mail (PEM)-encoded X.509 certificates, which are all concatenated together.
```

Comment:
I am thinking we should add a section here around _why_ a cluster owner would need to specify the "additional trust bundle" as an installation argument. There is a good explanation about the reasons why in the OCP product documentation (https://docs.openshift.com/container-platform/4.9/networking/configuring-a-custom-pki.html), maybe it should be repeated here:

> The additionalTrustBundle field is required unless the proxy’s identity certificate is signed by an authority from the RHCOS trust bundle. If you use an MITM transparent proxy network that does not require additional proxy configuration but requires additional CAs, you must provide the MITM CA certificate.

(this comment applies to both `rosa` and `osd`)

Reply:
Interestingly, this note seems to disappear in enterprise-4.10. I added it to my sections, but I just wanted to point out that it looks like this is going away unless I am totally misreading this.

Reply:
Thanks for pointing that out, that's interesting! I did some digging to find out more about this change (https://github.com/openshift/openshift-docs/pull/38550). The reasons it was removed from enterprise-4.10 documentation make sense, but don't apply to OSD or ROSA and how we're implementing the proxy configuration. So we can safely leave it in.